nibabel/tests/test_nifti1.py

```diff
@@ -359,7 +360,7 @@ def test_qform_sform(self):
             aff, code = out_meth(coded=True)
             assert_array_equal(aff, nice_aff)
             assert_equal(code, 2)
-            assert_array_equal(out_meth(), nice_aff) # non coded
+            assert_array_equal(out_meth(), nice_aff)  # non coded
             # Affine can also be passed if code == 0, affine will be suitably set
```

Comment:
Took the time to understand the test. How about replacing with:

    # Affine may be passed if code == 0, but will be set to None